ch09-lists-tuples-and-dictionaries/9a-cats-with-hats

```diff
@@ -0,0 +1,16 @@
+# At the start all cats are hatless.
+cats = [False]*100
```

Comment:
Poor cat with indice 0, that will be ignored through the whole game! 
But I agree it will make it easier in the rest of the code.

Reply:
Alternatively, as you proposed several solutions including one with a dictionary, we can suggest a solution with a dictionary.
It will avoid this trick.